Sources/Widgets/Widgets3D/ImplicitPlaneWidget/index.js

```diff
@@ -48,15 +48,31 @@ function widgetBehavior(publicAPI, model) {
     }
 
     model.lineManipulator.setWidgetOrigin(model.widgetState.getOrigin());
+    model.lineManipulator.setWidgetNormal(model.activeState.getNormal());
     model.planeManipulator.setWidgetOrigin(model.widgetState.getOrigin());
+    model.planeManipulator.setWidgetNormal(model.widgetState.getNormal());
```

Comment:
model.widgetState or model.activeState ?

Reply:
I'll just switch to using activeState. (It doesn't actually matter for this widget since they're both the same.)